modules/user/server-ts/social/password/index.js

```diff
@@ -1,6 +1,6 @@
+import AuthModule from '@module/authentication-server-ts/social/AuthModule';
```

Comment:
It is better to export/import `AuthModule` from `@module/authentication-server-ts` directly